hr_attendance_management/models/hr_attendance_day.py

```diff
@@ -295,10 +295,11 @@ def _compute_break_total(self):
                 att_day.break_ids.mapped('total_duration') or [0])
 
     @api.multi
-    @api.depends('paid_hours', 'due_hours', 'extra_hours_lost')
+    @api.depends('paid_hours', 'due_hours', 'extra_hours_lost', 'leave_ids')
     def _compute_extra_hours(self):
+        sick_leave = self.env.ref('hr_holidays.holiday_status_sl')
         for att_day in self:
-            if att_day.leave_ids.holiday_status_id.name == 'Sick Leaves':
+            if sick_leave in att_day.leave_ids.mapped('holiday_status_id'):
```

Comment:
Or this:
```suggestion
            if 'Sick Leaves' in att_day.leave_ids.mapped(
                    'holiday_status_id.name'):
```

Reply:
No this is less good because the name can change depending of the language of the user! It's always better to check equality at  the record value instead of names that can be translated!